.gitignore

```diff
@@ -71,3 +71,7 @@ Guardfile
 .vagrant/
 tags
 .byebug_history
+
+# testing cache
+/features/support/fixtures/req_cache
+/features/support/fixtures/cassettes
```

Comment:
these need to not be in .gitignore - otherwise our tests will spam 3rd party servers